src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingScriptOperation.cs

```diff
@@ -71,6 +73,7 @@ public override void Execute()
                 publishModel.ScriptItemsCollected += this.OnPublishModelScriptItemsCollected;
                 publishModel.ScriptProgress += this.OnPublishModelScriptProgress;
                 publishModel.ScriptError += this.OnPublishModelScriptError;
+                publishModel.AllowSystemObjects = true;
```

Comment:
>publishModel.AllowSystemObjects = true; [](start = 16, length = 39)

Please make sure this doesn't impact scripting a database by including system dependencies in the script.